lib/assets/connection_cell/main.js

```diff
@@ -563,12 +567,23 @@ export function init(ctx, info) {
       },
     },
 
+    watch: {
```

Comment:
We could also handle the `openenSecrets` here. The drawback is that the `update_field` will be triggered unnecessarily